bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java

```diff
@@ -170,7 +185,7 @@ public void mutate(final Mutation mutation) throws IOException {
     }
   }
 
-  private void doMutation(final Mutation mutation) throws RetriesExhaustedWithDetailsException {
+  private void doMutation(final Mutation mutation) {
```

Comment:
Master doesn't show the "throws RetriesExhaustedWithDetailsException".  I'm not sure why this is showing up in the diff.  Very strange.